capstone/backend/src/main/java/com/google/servlets/Item.java

```diff
@@ -34,12 +34,17 @@ public long getId(){
     return itemId;
   }
 
-  public float getPrice() {
+  public double getPrice() {
     return itemPrice;
   }
 
   public String getName() {
     return itemName;
   }
 
+  public boolean equals(Item otherItem) {
```

Comment:
There is always some boilerplate to create a class. Can you try to use Google AutoValue to generate this class for you? Reference: https://github.com/google/auto/blob/master/value/userguide/index.md

Same as Store class.

Reply:
I think I am going to change Store to make it so that the items can be in any order, so I am not sure if AutoValue will work here

Reply:
You should still be able to override equals() method by using AutoValue. At lease change this Item class to AutoValue?